### DISCUSSION TITLE
Reduce AuthManager `call` span level

### DISCUSSION THREAD

**Discussion by arcstur** [maxcountryman/axum-login#223] [General]
I saw that AuthManager's `call` function uses an `info_span!` inside it. I think that either it should be a debug_span, because usually libraries use debug level; or it could have a better name (or both).

I usually put my global log level at info and suddenly an empty "call" span was appearing and I couldn't know why.

**Comment by maxcountryman** [comment]
> I usually put my global log level at info and suddenly an empty "call" span was appearing and I couldn't know why.

What subscriber are you using?

Usually an empty span is not visible. (I believe log is an exception to this.) However, once an event is emitted within the scope of the span then it becomes visible.

If this is a side effect of log, I'd suggest using tracing directly if you can.
